Jenkinsfile

```diff
@@ -89,3 +131,116 @@ int ecrLogin(String aws_region) {
     String dockerLogin = sh (label: "Getting Docker login from ECR", script: ecrCommand, returnStdout: true).replace("-e none","") // some parameters that AWS provides and docker does not recognize
     return sh(label: "Logging in with Docker", script: dockerLogin, returnStatus: true)
 }
+
+
+String tfEnv(String tfEnvRepo="https://github.com/tfutils/tfenv.git", String tfEnvPath="~/.tfenv") {
```

Comment:
we are not going to deploy terraform with this pipeline, so the this function and the others below are not needed, please remove them.